ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/WorkloadCves.helpers.js

```diff
@@ -68,3 +68,69 @@ export function typeAndEnterResourceFilterValue(entityType, value) {
 export function selectEntityTab(entityType) {
     cy.get(selectors.entityTypeToggleItem(entityType)).click();
 }
+
+/**
+ * Clean up any existing watched images via API
+ */
+export function tryUnwatchAllImages() {
```

Comment:
What's the thought process behind the prefix of `try` vs. just saying `unwatchAllImages`?

Reply:
This is adapter from a similar function in Collections that attempts to delete a single collection by name, if it exists. Since in this case we want to unwatch all images, and not specifically by name, I think it makes more sense to remove the `try` portion of the function name.